tools/autograd/gen_inplace_or_view_type.py

```diff
@@ -335,6 +342,19 @@ def emit_inplace_or_view_body(fn: NativeFunctionWithDifferentiabilityInfo) -> Li
         api_name = sig_group.faithful_signature.name()
     else:
         api_name = sig_group.signature.name()
+    # Throws if a InplaceOrView kernel is hit without Autograd being disabled.
+    # There're two valid cases calling InplaceOrView:
```

Comment:
I think this comment is nice but it is a bit weird to have a comment that explains what this code is *not* doing :p